inspirehep/modules/theme/jinja2filters.py

```diff
@@ -745,3 +748,33 @@ def weblinks(description):
     if description:
         return 'Link to ' + description
     return 'Link to fulltext'
+
+
+@blueprint.app_template_filter()
+def jobs_similar(id):
```

Comment:
Design-wise, we should not allow ES queries in Jinja filters. On the other hand, that's what a lot of other filters do, so this too shall pass (but it will be mercilessly refactored in the future).